### PR TITLE
Commented all routes in App.js

### LINKED ISSUE #324: Frontend route group commenting
**Please describe what has to be done**
 Label the frontend app.js route sections. Just something simple like `{ \* Main site pages *\ }`, `{ \* Group pages *\ }`, etc. for each block of routes. Also make sure non seem out of place, rearranging as necessary.

### PR DESCRIPTION
**Pull request by kalpthakkar**
### Description of change(s), including why:

Commented all routes inside src/App.js
https://github.com/PollBuddy/PollBuddy/issues/324

### Type of change(s)

- [ ] Adding comments (non-breaking change which fixes an issue)

### Any work to still do or other concerns:

- (none)

### Closing issues:

- Closes #324

### CODE REVIEW

**Review by hbh7 — DISMISSED** [review]
Great, thanks for contributing!

**Comment by kalpthakkar** [comment]
I am getting this issue: 
Screenshot (22)

Can you help me to fix this issue so that I can make my first contribution on Github?

**Comment by hbh7** [comment]
I can't fix the issue per-se, but I can explain what it means.

With our repo, we've got a couple different automated systems that run when someone submits a pull request. One of these is our CI-CD pipeline. This deploys a live running isolated instance of code changes on our server. Another 2 workflows are linting ones for code style. 

An interesting thing about workflows is how they handle outside contributor submissions. In the past, the lint ones ran and the CI ones didn't. The CI ones won't succeed for security reasons as they require access to secrets stored in the repo. Essentially, the CI workflow is remote code execution so we want to keep access that restricted. Therefore, for the foreseeable future, the CI workflows will never work with external PRs. 

I don't know why the lint ones didn't run though, that's weird. I'll look into that. 

From your perspective though, you don't need to worry about any of this. The CI ones can fail and that's fine. In this case, the lint ones wouldn't have caught anything anyways, but still should have ran. So, no worries on your end but I will look into it. 

Our process for merging PRs is generally I (the project manager) will review every PR and then hopefully within a week of my review, someone else will have also left a review. If so then I'll merge shortly after that review. Otherwise, if the week timeout expires and I approved it, I'll merge it then. 

Basically, you're all set. Your PR should be merged within a week. 

Thanks for your submission, and I hope this helps with your learning! :) Let me know if you have any other questions with anything.

**Comment by kalpthakkar** [comment]
Thanks for the clarification :)
